images/fcos-bastion-image/root/usr/bin/reserve-hosts.sh

```diff
@@ -18,6 +18,7 @@ VENDOR_REGEX=""
 
 if [ -n "$VENDOR" ]; then
   VENDOR_REGEX=",${VENDOR},"
+  # shellcheck disable=SC2086
   echo $VENDOR_REGEX
```

Comment:
```suggestion
```

Let's just remove this line. @bmanzari do you confirm it was added only for debugging purposes?

Reply:
I run below script for testing:

cat test_vips.sh
#!/bin/bash

REQUEST_VIPS="true"
CANDIDATES_HOSTS=("host1,host2,1" "host3,host4,10" "host5,host6,255")
BUILD_DIR="."

# Script logic
if [ "$REQUEST_VIPS" == "true" ]; then
    FIRST_HOST_ID=$(echo "${CANDIDATES_HOSTS[0]}" | cut -f3 -d,)
    FIRST_HOST_ID_HEX=$(printf '%03x' "$FIRST_HOST_ID")

    INGRESS_VIP="${INGRESS_VIPS_SUBNET_PREFIX:-192.168.82.}${FIRST_HOST_ID}"
    API_VIP="${API_VIPS_SUBNET_PREFIX:-192.168.81.}${FIRST_HOST_ID}"
    INGRESS_VIP_V6="${IPV6_INGRESS_VIPS_SUBNET_PREFIX:-fd99:2222:3456::2:}${FIRST_HOST_ID_HEX}"
    API_VIP_V6="${IPV6_API_VIPS_SUBNET_PREFIX:-fd99:2222:3456::1:}${FIRST_HOST_ID_HEX}"

    # Create a single JSON object
    echo "{
  \"ingress_vip\": \"$INGRESS_VIP\",
  \"api_vip\": \"$API_VIP\",
  \"ingress_vip_v6\": \"$INGRESS_VIP_V6\",
  \"api_vip_v6\": \"$API_VIP_V6\"
}" > "${BUILD_DIR}/vips.json"
fi

The result is:
cat vips.json
{
  "ingress_vip": "192.168.82.1",
  "api_vip": "192.168.81.1",
  "ingress_vip_v6": "fd99:2222:3456::2:001",
  "api_vip_v6": "fd99:2222:3456::1:001"
}